test/evergreen.yml

```diff
@@ -2069,7 +2069,7 @@ tasks:
         params:
           working_dir: "wiredtiger/cmake_build"
           shell: bash
-          silent: false
+          silent: true
```

Comment:
Is `silent` is `true`, I am not sure if `set -o verbose` does anything?

Reply:
That is okay for now, we want to show logs from the testing without exposing credentials. So this will go back to being silent false, but with a different method to set the environment variables. We have WT-8839 for this.